sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/Options.cs

```diff
@@ -20,9 +14,6 @@ public class Options
         [Option('i', "clientId", Required = true, HelpText = "Client Id of the application Id to login, or the application Id used to log the user in.")]
```

Comment:
we could make these optional going forward if we want to support different login methods using the same DefaultAzureCredential. At that point, we could stop overriding the environment variables.